src/main/java/frc/team4909/team4909SwerveModule.java

```diff
@@ -0,0 +1,51 @@
+package frc.team4909;
+
+import frc.bionic.swerve.*;
+
+public class team4909SwerveModule extends frc.bionic.swerve.SwerveModule{
```

Comment:
This is already in directory and package called team4909, so doesn't need the extra prefix on the class name or file name.